### PR TITLE
Do not show Not User for admin support user for AB#16295

### PR DESCRIPTION
**Pull request by BrianMaki**
# Fixes [AB#16295](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/16295)

## Description

- Do not show 'Not User' warning message to admin support user
- Added and updated ui patient details functional tests including removing no longer needed Not Found scenario
- Consolidated fixtures i.e., removed unnecessary ones and merged ones that could be merged
- Added intercepts in patient details ui tests to improve performance
- Removed unused MessagingVerification fixture

**Admin User:**

Screenshot 2023-10-17 at 12 14 55 PM

**Support User:**

Screenshot 2023-10-17 at 12 14 30 PM

**Patient Details ui:**

Screenshot 2023-10-17 at 2 52 07 PM

**Patient Details e2e**

Screenshot 2023-10-17 at 2 51 55 PM

**Support ui:**

Screenshot 2023-10-17 at 2 51 43 PM

## Testing

- [ ] Unit Tests Updated
- [x] Functional Tests Updated
- [ ] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs [hunk 1]
```diff
@@ -92,6 +92,8 @@ public partial class PatientDetailsPage : FluxorComponent
 
         private string? StatusWarning => this.Patient == null ? null : FormattingUtility.FormatPatientStatus(this.Patient.Status);
 
+        private bool ShowStatusWarning => !(this.UserHasRole(Roles.Support) && this.Patient?.Status == PatientStatus.NotUser);
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs:95]: I think this is more understandable if the brackets are done away with, e.g.

`this.UserHasRole(Roles.Admin) || this.UserHasRole(Roles.Reviewer) || this.Patient?.Status != PatientStatus.NotUser`

It also better handles the case where a user has both the Admin and Support roles.

Comment (BrianMaki) [reply Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs:95]: Does that handle when user has AdminUser role and the status is NotUser?

Comment (MikeLyttle) [reply Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs:95]: Admin and Reviewer role can see the status warning.  Other roles can see it only when the status is something other than "Not User".

Comment (BrianMaki) [reply Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs]: Changed.

**Review by MikeLyttle — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Review by BronzBierd — APPROVED** [review]

**Pull request merged**